gateway-ha/src/main/java/io/trino/gateway/ha/router/QueryCountBasedRouter.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.gateway.ha.router;
+
+import io.trino.gateway.ha.clustermonitor.ClusterStats;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.stream.Collectors;
+
+public class QueryCountBasedRouter
```

Comment:
Fix the indentation in this class.

Reply:
not sure, what's the issue here, can you elaborate? I rely on the check-style to guide me.

Reply:
> I rely on the check-style to guide me.

Are you really using Airlift code style? https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style

The indention at L30 & L72-74 are wrong.
```java
public class QueryCountBasedRouter
            extends StochasticRoutingManager
↓
public class QueryCountBasedRouter
        extends StochasticRoutingManager
```

```java
            filteredList = clusterStats.stream()
                .filter(stats -> stats.healthy())
                .filter(stats -> routingGroup.equals(stats.routingGroup()))
                .collect(Collectors.toList());
↓
            filteredList = clusterStats.stream()
                    .filter(stats -> stats.healthy())
                    .filter(stats -> routingGroup.equals(stats.routingGroup()))
                    .collect(Collectors.toList());
```

Reply:
Yes, I am using the airbase check-style. The checks on the PR are also not catching these error.

Reply:
The current check-style doesn't seem to throw exception https://github.com/airlift/airbase/blob/ac2bd5fe5a9cbfe351096906fe729dcb6aeab460/airbase-policy/src/main/resources/checkstyle/airbase-checks.xml#L156
Not sure if there's a way to enforce the indentation to let check-style fail if errors like this occur again going forward